docs/en/latest/how-to-build.md

```diff
@@ -76,6 +76,8 @@ Please refer to: [Installing Apache APISIX with Helm Chart](https://github.com/a
   cd apisix-2.9
   # Create dependencies
   make deps
+  # Install apisix command
+  make install
```

Comment:
Please update [CN version](https://github.com/apache/apisix/blob/master/docs/zh/latest/how-to-build.md) at the same time.

Reply:
I think `make install` means installing the APISIX code into luarocks. The context here is to download the APISIX source code, so does the source code still need to be installed to luarocks? I thought it was no longer needed. what's your option? @spacewander @membphis

Reply:
Yes. See the comment here:
https://github.com/apache/apisix/blob/6162ca0ef92445fdd15b7a213655887b66cd4eea/Makefile#L167

Reply:
I changed my mind. Look like we still need this command to install APISIX to PATH even `make install` is designed for luarocks.

Reply:
why

Reply:
As there is no other command to do it.